tools/export-generate/main.go

```diff
@@ -124,12 +128,12 @@ func main() {
 				log.Fatalf("unable to decode key: %v", k.Key)
 			}
 			exposureKeys[i].ExposureKey = decoded
-			n, err := util.RandomIntervalCount()
-			if err != nil {
-				log.Fatalf("problem with interval count: %v", err)
+			exposureKeys[i].IntervalNumber = k.IntervalNumber - 144 // typically the key will be at least 1 day old
```

Comment:
nit: instead of 144 we could make use of the MaxIntervalCount constant in publish/model/exposure_model.go

Reply:
@gurayAlsac Ok, I [changed](https://github.com/google/exposure-notifications-server/pull/426/commits/91cd84361bd9b953bb28408bddbfcc33104d9223) that.